apache/superset
# feat: Move Database Import option into DB Connection modal

### SUMMARY

This PR moves the import database from file option inside of the database connection modal.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

#### BEFORE:
<img width="217" alt="importBefore" src="https://user-images.githubusercontent.com/55605634/160602837-e6196ecc-1ae5-44d2-a71b-5160a59a382b.png">
<img width="749" alt="passwordBefore" src="https://user-images.githubusercontent.com/55605634/160602859-4f75a771-8b13-4bc6-8436-7a22a946b0b1.png">
<img width="749" alt="overwriteBefore" src="https://user-images.githubusercontent.com/55605634/160602885-746751fb-a4de-40c5-9623-4f9fc59c2f10.png">

#### AFTER:
<img width="500" alt="importAfter" src="https://user-images.githubusercontent.com/55605634/160604167-3a1ad3b6-4c1b-4410-b441-63f1ecaa1902.png">
<img width="501" alt="passwordAfter" src="https://user-images.githubusercontent.com/55605634/160603040-afebf327-aa68-4704-a0d1-4842304203df.png">
<img width="501" alt="overwriteAfter" src="https://user-images.githubusercontent.com/55605634/160603057-dfea78e7-96fd-4b89-a877-da4abb4ce6e1.png">
<img width="500" alt="pwowAfter" src="https://user-images.githubusercontent.com/55605634/160717065-7c466d90-b8cc-4985-b1a8-3ac82db4a924.png">


### TESTING INSTRUCTIONS

- Open the database connection modal and click "Import database from file" at the bottom
- Import a database from file that requires a password to see the new password confirmation
- Import a database from file that already exists to see the new overwrite confirmation
- If you import a database that doesn't already exist and doesn't require a password, the modal should close and add the database with no further confirmation.

### ADDITIONAL INFORMATION
